engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -680,7 +680,7 @@ class Compiler(
     * Given module A/B/C.enso
     * ````
     *   type C
-    *       type C a
+    *       C a
```

Comment:
If we can catch this now, then great!

Reply:
This documentation was using the outdated syntax